### PR TITLE
Fetch employee events for each month.

### PR DESCRIPTION
**Pull request by kirsty-unosquare**
Updated fetching event call for calendar. Also updated some terms to match new event call. Let me know if you see anywhere that needs updated. Very similar to web app.

Had to use the holidays call for profile screen since I need to retrieve all events. Currently the profile screen is broken but will fix on another branch.

### CODE REVIEW

**Review by William-Unosquare — APPROVED** [review]
All looks good to me. :)

**Review by d1wilko — APPROVED** [review]
nothing to call out - nice code

File: NativeApp/src/components/User/View/index.js [hunk 1]
```diff
@@ -27,14 +27,14 @@ const UserView = (props) => {
...
-          takenHolidays={takenHolidays}
+          events={events}
           remainingHolidays={remainingHolidays}
         />
         <ScrollView>
           <View style={styles.flatListView}>
             <FlatList
-              keyExtractor={item => item.holidayId.toString()}
-              data={takenHolidays}
+              keyExtractor={(item, index) => index.toString()}
```
Comment (paulmatchett) [inline NativeApp/src/components/User/View/index.js]: Can you not use eventId instead of index?

**Review by paulmatchett — APPROVED** [review]
Nice and neat. All looks good to me

File: NativeApp/src/components/User/View/index.js (hunk 1 above)
Comment (kirsty-unosquare) [reply NativeApp/src/components/User/View/index.js]: Actually need to convert this back  to holiday id. The user screen still uses the 
 Get `/holidays/findByEmployeeId/{employeeId}` endpoint. I don't think eventId is listed in the response for this request.

Reverting back to holidayId, good spot!

**Pull request merged**